src/components/Navigation/NavigationTabBar/index.tsx

```diff
@@ -127,22 +99,9 @@ function NavigationTabBar({selectedTab, isTooltipAllowed = false, isTopLevelBar
             return;
         }
         interceptAnonymousUser(() => {
-            const settingsTabState = getSettingsTabStateFromSessionStorage();
-            if (settingsTabState && !shouldUseNarrowLayout) {
-                const stateRoute = findFocusedRoute(settingsTabState);
-                if (!subscriptionPlan && stateRoute?.name === SCREENS.SETTINGS.SUBSCRIPTION.ROOT) {
-                    Navigation.navigate(ROUTES.SETTINGS_PROFILE.route);
-                    return;
-                }
-                const lastVisitedSettingsRoute = getLastVisitedTabPath(settingsTabState);
-                if (lastVisitedSettingsRoute) {
-                    Navigation.navigate(lastVisitedSettingsRoute);
-                    return;
-                }
-            }
-            Navigation.navigate(ROUTES.SETTINGS);
+            navigationRef.dispatch({type: CONST.NAVIGATION.ACTION_TYPE.PUSH, payload: {name: NAVIGATORS.SETTINGS_SPLIT_NAVIGATOR}});
```

Comment:
Let's add a comment that initial params are set during preload. It's not obvious